providers/pool.rb

```diff
@@ -245,3 +270,11 @@ def configure
     end
   end
 end
+
+private
+def configure_application_pool(condition, config, add_remove = '')
+  if(condition)
+    @was_updated = true
+    @cmd << " \"/#{add_remove}[name='#{@new_resource.pool_name}'].#{config}\""
```

Comment:
We never had that before ever.  I did add a lot of commands so I'm not 100 on this

Justin Schuhmann

> On Feb 12, 2015, at 12:09 PM, Kartik Null Cating-Subramanian <notifications@github.com> wrote:
> 
> In providers/pool.rb:
> 
> > @@ -245,3 +270,11 @@ def configure
> >      end
> >    end
> >  end
> > +
> > +private
> > +def configure_application_pool(condition, config, add_remove = '')
> > +  if(condition)
> > +    @was_updated = true
> > +    @cmd << " \"/#{add_remove}[name='#{@new_resource.pool_name}'].#{config}\""
> The IIS docs keep mentioning the need for "/commit:apphost" for a number of commands. I'm not sure of what the significance is. Do we need to add that here?
> 
> —
> Reply to this email directly or view it on GitHub.
>